registration-processor-default.properties

```diff
@@ -483,14 +484,14 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.version=0.9
 mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1=${mosip.biosdk.default.service.url}
 
 ## Credential requestor
-mosip.registration.processor.credentialtype=euin
-mosip.registration.processor.encrypt=false
-mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator 
+#mosip.registration.processor.credentialtype=euin
+#mosip.registration.processor.encrypt=false
+#mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator
```

Comment:
Remove these duplicate properties and put correct values in the properties where ever it is mentioned.

Reply:
removed